### ISSUE TITLE
Create Organization - County for Org Location

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#2210]
Should this be a pre-populated dropdown of countries to avoid discrepancies of user input?

**Comment by Jackyckyau** [comment]
Agreed. We should populate a list of countries, which can help to keep the same format

**Comment by thomfresn** [comment]
There's a [REST API](https://github.com/fayder/restcountries) that provides a list of countries. Or the list can be hardcoded based on the [ISO country code list](https://www.iso.org/obp/ui/#search).
